### DISCUSSION TITLE
Videos for Cobbler Demos

### DISCUSSION THREAD

**Discussion by SchoolGuy** [cobbler/cobbler#3151] [General]
I was asked numerous times now about videos for Cobbler. Since they require a great deal of work, I was hoping to get topics the community is interested in. Depending on the demand and effort I have to honestly say that I am thinking about only making this available to Sponsors but this is thing that is not set in stone yet.

TODOs for videos:

- [ ] What is Cobbler?
- [ ] How to install Cobbler?
- [ ] First Steps with Cobbler
- [ ] How to execute a sync and what is a `cobbler sync`?
- [ ] What is `cobbler import` and how to use it?

TODOs for [asciinema](https://asciinema.org/):

- [ ] Update Installation recording
- [ ] How to use the `scm_track` integration

TODOs for blog posts:

- [ ] How to deploy ESXI and Proxmox with Cobbler?
- [ ] How to deploy systems with Cobbler in ESXI and Proxmos via Cobbler?

**Comment by zparihar** [comment]
I think Videos for Cobbler would be great.  It would make it easier for more people to easily enter this community.

I like the idea of: 
- Installing Cobbler
- Setting up Cobbler to do PXE Boot for AlmaLinux
- How to use the `scm_track` integration

Cheers!
